Builds/DockerPlay/README.md

```diff
@@ -0,0 +1,157 @@
+# DockerPlay : what is this ?
+
+As a lucky Debian Jessie users some of us are missing an impotant stack to play with rippled from the sources : gcc5. Instead of hacking our working station (and potentially break it) we can setup a rippled development environment thanks docker. 
```

Comment:
It would be great if this could be reworded to be a little more formal. Perhaps:

> Setup a working `rippled` development and testing environment inside a Docker container, without the need to install tools and libraries (such as `gcc5`) on Debian Jessie. 

I'm not very familiar with Docker, and I'm curious, will this particular Docker build work _only_ on Debian Jessie hosts? How easy would it be for this container to run, say, on other Linux distributions, such as Ubuntu, Fedora, CentOS or RHEL hosts?